tools/basic-illumination/.shed.yml

```diff
@@ -0,0 +1,6 @@
+owner: perssond
```

Comment:
Did you all agreed on one common user for this repo?

Reply:
We tried but ran into some issues with changing owner for repos that already exist in the toolshed. So reverting back to the original owner with the lab user have admin access on the repo.